bin/inference/pycbc_inference_plot_movie

```diff
@@ -116,6 +119,14 @@ parser.add_argument('--dpi', type=int, default=200,
 parser.add_argument("--nprocesses", type=int, default=None,
                     help="Number of processes to use. If not given then "
                          "use maximum.")
+parser.add_argument("--make-movie", type=str,
```

Comment:
This is a bad name for what this option means. Should be like ``--movie-file`` or something.

Same with ``--output-file``. Its not an output file. Should be like ``--output-dir``.